packages/app-accounts/src/index.tsx

```diff
@@ -12,8 +12,14 @@ import { HelpOverlay, Tabs } from '@polkadot/react-components';
 
 import basicMd from './md/basic.md';
 import Overview from './Overview';
+import AddressBook from '../../app-address-book/src/Overview'
 import { useTranslation } from './translate';
-import Vanity from './Vanity';
+import {SubjectInfo} from "@polkadot/ui-keyring/observable/types";
+
+interface Props extends AppProps {
+  allAddresses?: SubjectInfo;
```

Comment:
Where are these fields used?

Reply:
they are passed down as props to address book https://github.com/cennznet/ui/blob/651a9d574ee6d5c6b05c408504698aed7ef322de/packages/app-address-book/src/Overview.tsx